koku/masu/external/downloader/ocp/ocp_report_downloader.py

```diff
@@ -51,7 +51,12 @@ def divide_csv_daily(file_path, filename):
     daily_files = []
     directory = os.path.dirname(file_path)
 
-    data_frame = pd.read_csv(file_path)
+    try:
+        data_frame = pd.read_csv(file_path)
+    except Exception as error:
+        LOG.warning(f"File {file_path} could not be parsed. Reason: {str(error)}")
```

Comment:
If we're raising the error, this should probably be LOG.error

Reply:
:facepalm: What was I thinking when I put  `LOG.warning`. Fixing this. Thanks.